addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -212,11 +212,9 @@ export class WebsocketWorker {
      */
     _addChannel(client, channel) {
         const clientChannels = this.channelsByClient.get(client);
-        if (!clientChannels.includes(channel)) {
-            clientChannels.push(channel);
-            this.channelsByClient.set(client, clientChannels);
-            this._debouncedUpdateChannels();
-        }
+        clientChannels.push(channel);
+        this.channelsByClient.set(client, clientChannels);
+        this._debouncedUpdateChannels();
```

Comment:
Shouldn't the update only be done if `if (!clientChannels.includes(channel)) {` initially to avoid unnecessary call?
Or is that already handled internally?